.circleci/config.yml

```diff
@@ -589,16 +589,17 @@ workflows:
       - test-driver:
           matrix:
             parameters:
-              driver: ["bigquery-cloud-sdk", "googleanalytics"]
+              driver: ["bigquery-cloud-sdk", "googleanalytics", "redshift", "athena"]
           name: be-tests-<< matrix.driver >>-ee
           requires:
-            - be-tests-java-11-ee
```

Comment:
Not really needed for this PR but by running all the driver tests as soon as we finish fetching deps (instead of waiting for regular Java 11 tests to finish) we should speed up CI a bit. The main reason I had them waiting for Java 11 tests in the first place was to avoid running a bunch of tests that will fail for sure and tying up our limited concurrency; that's not really an issue any more since we're on a different plan, so we should take advantage of that to get quicker feedback.